### DISCUSSION TITLE
Option to automatically move completed cards to bottom of column

### DISCUSSION THREAD

**Discussion by samwarnick** [mgmeyers/obsidian-kanban#478] [Ideas & Feature Requests]
I would love it if there was an option to move a completed card to the bottom of the list when checked, and then moved back up to the bottom of the incomplete tasks if unchecked. Is there already a way to do this? This could probably be generalized with some sorting options.

**Comment by dayeggpi** [comment]
additionally, if there is a way to move a completed card into the first column that has the tag "mark cards in this list as completed" active... (i.e. we have two columns "In progress" and "Done". "Done" has the tag "mark cards in this list as completed" active. I have a card in "In progress" and mark it as completed, then the card moves to column "Done" automatically)
